packages/chakra-ui/src/AltDateWidget/AltDateWidget.tsx

```diff
@@ -16,6 +19,31 @@ const rangeOptions = (start: number, stop: number) => {
   return options;
 };
 
+function DateElement<
```

Comment:
This componente was originally created within the AltDateWidget component, so I extracted it to prevent potential performance issues where the component may be excessively re-rendered.

Reply:
I wonder if we should do this for other themes?